src/style.css

```diff
@@ -0,0 +1,67 @@
+input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+}
+
+.icon {
+  background: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.icon-big {
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+}
+
+.icon-small {
+  background-position: 13px 15px;
+  background-size: 11px 11px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.